packages/siop-oid4vp/lib/op/OPBuilder.ts

```diff
@@ -13,7 +13,7 @@ export class OPBuilder {
   expiresIn?: number
   issuer?: IIssuerId | ResponseIss
   responseMode?: ResponseMode = ResponseMode.DIRECT_POST
-  responseRegistration?: Partial<ResponseRegistrationOpts> = {}
+  responseRegistration?: Partial<ResponseRegistrationOpts> //= {}
```

Comment:
This is likely to cause issues in certain situations as the builder properties are assigned to the OP in the build phase. So why was this needed?

Reply:
I probably did this when fighting sudden issues with LanguageTagUtils.
```
    const languageTaggedFields: Map<string, string> = LanguageTagUtils.getLanguageTaggedProperties(
      opts.builder.responseRegistration,
      languageTagEnabledFieldsNames,
    )
```
when opts.builder.responseRegistration is undefined it's not going to try to translate. When an empty object it's going to throw error because of missing fields.

They also pass in languageTagEnabledFieldsNames which was actually interpreted as languageTagRequiredFieldsNames. But I think I fixed that at a later point, so I will try to revert that.